### ISSUE TITLE
handling if / else if / else

### ISSUE THREAD

**Issue by RandomEtc** [baryshev/ect#9]
I get "unexpected indent" errors for an if/else if/else statement. Something like this:

```
<% if 'one thing': %>
foo
<% else if 'other': %>
bar
<% else: %>
baz
<% end %>
```

Should that work?

**Comment by baryshev** [comment]
Thanks for reporting. Error resolved in version 4.3. Already available via npm.

**Comment by RandomEtc** [comment]
Great - thanks!

**Comment by Naatan** [comment]
Seems this issue is back? Reproducible on ectjs.com:

```
<% unless @id: %>
foo 
<% else if @id: %>
bar
<% end %>
```

**Comment by baryshev** [comment]
Thank you for reporting. Fixed in `v0.5.9`.
